cpu/atmega1281/ldscripts_compat/README.md

```diff
@@ -6,3 +6,9 @@ overriden to configure ROM and RAM length.
 
 This ldscript is used for supporting older versions on avr-binutils like on
 ubuntu xenial.
+
+Generated by extracting the output of
+
+    avr-gcc -Wl,--verbose --mmcu=$(CPU)
```

Comment:
I get an error for this command, need to use `-mmcu` instead of `--mmcu` (no double dashes)

what I ran is 
```shell
avr-gcc -Wl,--verbose -mmcu=atmega1281
```

I have avr-binutils 2.32 and avr-gcc 8.3.0 (on macOS installed via brew)

Reply:
Indeed, was a mistake in the commit message too now that I remember.
My bad for not re-testing…

Reply:
It is bad to rush these changes ><